mPower2/mPower2/Triggers.json

```diff
@@ -0,0 +1,118 @@
+{
+    "identifier": "Triggers",
```

Comment:
Weren't we going to change the name of Triggers to something else? seems like there was a JIRA for that (but maybe this is just a copypaste or moving an existing file and the Triggers rename will happen in a separate PR, in which case just ignore this)

Reply:
Um... I don't remember that, though yes, this is a copy/paste.